easybuild/easyconfigs/b/BLAST+/BLAST+-2.9.0-foss-2019a.eb

```diff
@@ -0,0 +1,51 @@
+##
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>, Kenneth Hoste (UGent)
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of
+# the policy: http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'BLAST+'
+version = '2.9.0'
+
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/%(namelower)s/%(version)s/']
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+checksums = ['a390cc2d7a09422759fc178db84de9def822cbe485916bbb2ec0d215dacdc257']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('PCRE', '8.43'),
+    ('Boost', '1.67.0'),  # build fails with Boost-1.70.0
```

Comment:
Introducing another `Boost` version in the `2019a` generation of easyconfigs is going to be a PITA...

How bad does the build fail? Is it fixable?

Reply:
All failures are related to Boost.Test use:

* `collector` changed name to `collector_t`,
* implementation detail `ignore_unused_variable_warning` moved to Boost.Core's `ignore_unused`,
* `log_build_info` grew a new argument with a default parameter.

A standalone build completes and passes `make check` with a patch that addresses these three problems.

Reply:
challenge accepted :)

Reply:
:+1: